test/e2e/pd.go

```diff
@@ -35,22 +35,20 @@ import (
 
 var _ = Describe("Pod Disks", func() {
 	var (
-		c         *client.Client
 		podClient client.PodInterface
 		host0Name string
 		host1Name string
 	)
+	framework := Framework{BaseName: "pod-disks"}
 
 	BeforeEach(func() {
-		var err error
-		c, err = loadClient()
-		expectNoError(err)
-
 		SkipUnlessNodeCountIsAtLeast(2)
 
-		podClient = c.Pods(api.NamespaceDefault)
+		framework.beforeEach()
```

Comment:
Yes. You can look up ginkgo docs to verify, but I believe they're called in the order they're declared.